lib/Controller/AdminController.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (c) 2023, MetaProvide Holding EKF
+ *
+ * @author Ron Trevor <ecoron@proton.me>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Files_External_Ethswarm\Controller;
+
+use OCP\AppFramework\Controller;
+use OCP\IConfig;
+use OCP\IRequest;
+use OCA\Files_External_Ethswarm\Settings\Admin;
+use OCP\AppFramework\Http\DataResponse;
+use OCP\AppFramework\Http;
+
+class AdminController extends Controller {
+
+	/** @var Admin */
+	private $adminmapper;
```

Comment:
I think it would be better to just keep the name `admin` as adding mapper usually indicates DB related activity.

Reply:
Corrected